### PR TITLE
JAM STANAG and 7.62.x51

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- Make changes to STANAG mags as discussed here https://github.com/CBATeam/CBA_A3/issues/108#issuecomment-425699133 lower
- Add long and drum mag classes for 7.62x51mm
- add 5.56 FAMAS F1 magwell
* get Dro a tshirt, ref https://github.com/CBATeam/CBA_A3/pull/995#issuecomment-426336948

### CODE REVIEW

**Comment by commy2** [comment]
Sorry, Drofseh. I couldn't tell.

**Comment by Drofseh** [comment]
Ok, I'm happy with this if @robalo is.

**Comment by robalo** [comment]
Spotted some missing ;

File: addons/jam/magwells_65G.hpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
-    class CBA_65G_STANAG {};            // 6.5mm Grendel in a normal STANAG mag
-    class CBA_65G_STANAG_L {};          // 6.5mm Grendel in a long STANAG mag
-    class CBA_65G_STANAG_XL {};         // 6.5mm Grendel in a drum or XL stick mag
-    class CBA_65G_STANAG_2D {};         // 6.5mm Grendel in a twin-drum mag
-    class CBA_65G_STANAG_2D_XL {};      // 6.5mm Grendel in a XL twin-drum mag
+    class CBA_65G_STANAG {};            // 6.5mm Grendel in a normal length STANAG mag, including small drums
+    class CBA_65G_STANAG_L {};          // 6.5mm Grendel in a long STANAG stick or coffin
+    class CBA_65G_STANAG_XL {};         // 6.5mm Grendel in an extra long STANAG stick or coffin mag
+    class CBA_65G_STANAG_2D {};         // 6.5mm Grendel in a twin-drum STANAG mag
+    class CBA_65G_STANAG_2D_XL {}       // 6.5mm Grendel in an extra large twin-drum STANAG mag
```
Comment (commy2) [inline addons/jam/magwells_65G.hpp]: ;

File: addons/jam/magwells_68SPC.hpp [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
-    class CBA_68SPC_STANAG_2D {};       // 6.8 SPC in a twin-drum mag
-    class CBA_68SPC_STANAG_2D_XL {};    // 6.8 SPC in a XL twin-drum mag
-    class CBA_68SPC_STANAG_L {};        // 6.8 SPC in a long STANAG mag
-    class CBA_68SPC_STANAG_XL {};       // 6.8 SPC in a drum or XL stick mag
-    class CBA_68x43_ACR {};             // 6.8mm Remington SPC
+    class CBA_68SPC_STANAG {};          // 6.8 SPC in a normal length STANAG mag, including small drums
+    class CBA_68SPC_STANAG_L {};        // 6.8 SPC in a long STANAG stick or coffin
+    class CBA_68SPC_STANAG_XL {};       // 6.8 SPC in an extra long STANAG stick or coffin mag
+    class CBA_68SPC_STANAG_2D {};       // 6.8 SPC in a twin-drum STANAG mag
+    class CBA_68SPC_STANAG_2D_XL {}     // 6.8 SPC in an extra large twin-drum STANAG mag
```
Comment (commy2) [inline addons/jam/magwells_68SPC.hpp]: ; @ EOL

File: addons/jam/magwells_762x39.hpp [hunk 3]
```diff
@@ -9,8 +9,8 @@
...
-    class CBA_762x39_STANAG {};         // 762x39mm in a normal STANAG mag
-    class CBA_762x39_STANAG_L {};       // 762x39mm in a long STANAG mag
-    class CBA_762x39_STANAG_XL {};      // 762x39mm in a drum or XL stick mag
-    class CBA_762x39_STANAG_2D {};      // 762x39mm in a twin-drum mag
-    class CBA_762x39_STANAG_2D_XL {};   // 762x39mm in a XL twin-drum mag
+    class CBA_762x39_STANAG {};         // 762x39mm in a normal length STANAG mag, including small drums
+    class CBA_762x39_STANAG_L {};       // 762x39mm in a long STANAG stick or coffin
+    class CBA_762x39_STANAG_XL {};      // 762x39mm in an extra long STANAG stick or coffin mag
+    class CBA_762x39_STANAG_2D {};      // 762x39mm in a twin-drum STANAG mag
+    class CBA_762x39_STANAG_2D_XL {}    // 762x39mm in an extra large twin-drum STANAG mag
```
Comment (commy2) [inline addons/jam/magwells_762x39.hpp]: ; @ EOL

File: addons/jam/magwells_65G.hpp (hunk 1 above)
Comment (Drofseh) [reply addons/jam/magwells_65G.hpp]: oh lordy I am blind

File: addons/jam/magwells_300BLK.hpp [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
-    class CBA_300BLK_STANAG_L {};       // .300 Blackout in a long STANAG mag
-    class CBA_300BLK_STANAG_XL {};      // .300 Blackout in a drum or XL stick mag
-    class CBA_300BLK_STANAG_2D {};      // .300 Blackout in a twin-drum mag
-    class CBA_300BLK_STANAG_2D_XL {};   // .300 Blackout in a XL twin-drum mag
-    class CBA_762x35_STANAG {};         // .300 Blackout
+    class CBA_300BLK_STANAG {};         // .300 Blackout in a normal length STANAG mag, including small drums
+    class CBA_300BLK_STANAG_L {};       // .300 Blackout in a long STANAG stick or coffin
+    class CBA_300BLK_STANAG_XL {};      // .300 Blackout in an extra long STANAG stick or coffin mag
+    class CBA_300BLK_STANAG_2D {};      // .300 Blackout in a twin-drum STANAG mag
+    class CBA_300BLK_STANAG_2D_XL {}    // .300 Blackout in an extra large twin-drum STANAG mag
```
Comment (commy2) [inline addons/jam/magwells_300BLK.hpp]: ; @ EOL

**Comment by commy2** [comment]
Now the indentation of the comments isn't pretty anymore. :(

**Comment by Drofseh** [comment]
@commy2 all done

**Comment by Drofseh** [comment]
Ok, comments are aligned.

**Comment by robalo** [comment]
One more thing, initially I didn't allow the largest twin drums on FN F2000 and Tavors. This commit allows them. I thought my setup was an OK compromise.

**Comment by Drofseh** [comment]
My bad, I thought they were on all the vanilla rifles originally.
I think your reasoning is correct and I will change them back.

**Comment by robalo** [comment]
Good work. Go JAM !

**Pull request merged**